### PR TITLE
Add build for 2023.2 Bobcat

### PR DESCRIPTION
**Pull request by osfrickler**

### CODE REVIEW

File: patches/2023.2/kolla/build-ovn-from-sources.patch [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
++        libssl-dev \
++        libcap-ng-dev \
++        libunbound-dev \
++        libjemalloc-dev \
++        libunbound8 \
++        iproute2 \
++        jq \
++        libjemalloc2 \
++        procps \
++    && git clone -b v23.06.2 --depth 1 https://github.com/ovn-org/ovn.git /ovn \
```
Comment (berendt) [inline patches/2023.2/kolla/build-ovn-from-sources.patch]: Use v23.09.1 here

File: templates/2023.2/template-overrides.mako [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+LABEL "build-date"="{{ build_date }}" ${"\\"}
+      "name"="{{ image_name }}" ${"\\"}
+      "de.osism.commit.docker_images_kolla"="${hash_docker_images_kolla}" ${"\\"}
+      "de.osism.commit.kolla"="${hash_kolla}" ${"\\"}
+      "de.osism.commit.kolla_version"="${kolla_version}" ${"\\"}
+      "de.osism.commit.release"="${hash_release}" ${"\\"}
+      "de.osism.release.openstack"="${openstack_version}" ${"\\"}
+      "de.osism.version"="${version}" ${"\\"}
+      "org.opencontainers.image.created"="${created}" ${"\\"}
+      "org.opencontainers.image.documentation"="https://docs.osism.tech" ${"\\"}
```
Comment (berendt) [inline templates/2023.2/template-overrides.mako]: Use https://osism.github.io as docs url (we've to change this for the 2023.1 builds as well).

**Review by berendt — CHANGES_REQUESTED** [review]

File: patches/2023.2/kolla/build-ovn-from-sources.patch (hunk 1 above)
Comment (osfrickler) [reply patches/2023.2/kolla/build-ovn-from-sources.patch]: done

File: templates/2023.2/template-overrides.mako (hunk 2 above)
Comment (osfrickler) [reply templates/2023.2/template-overrides.mako]: done

File: .zuul.yaml [hunk 3]
```diff
@@ -144,6 +151,19 @@
         secret: SECRET_CONTAINER_IMAGES_KOLLA
         pass-to-parent: true
 
+- job:
+    name: container-images-kolla-push-bobcat
+    parent: container-images-kolla-build
+    semaphores:
+      - name: semaphore-container-images-kolla-push-antelope
```
Comment (artificial-intelligence) [inline .zuul.yaml]:
```suggestion
      - name: semaphore-container-images-kolla-push-bobcat
```
?

Comment (berendt) [reply .zuul.yaml]: We also have to add this semaphore.

Comment (osfrickler) [reply .zuul.yaml]: updated

**Review by berendt — APPROVED** [review]

**Pull request merged**